noxfile.py

```diff
@@ -23,11 +24,24 @@
 
 REQUIRED_DOCKER_VERSION = "20.10.17"
 
-# Sets the default session to `--list`
-nox.options.sessions = []
```

Comment:
aw I kind of liked this functionality. I didn't know about `--list` before though, that's good to know!

Reply:
I added this note in the new table of `nox` commands. `nox -l` is the new way to get the list!